documentation/tutorial.md

```diff
@@ -0,0 +1,65 @@
+# Git Town Tutorial
+
+Let's assume you are part of a development team with Bob and Carol. Your repository is hosted on GitHub, and you follow [GitHub Flow](https://guides.github.com/introduction/flow/index.html).
+
+
+## Starting a new feature
+
+You are in the middle of the sprint, and have just finished a feature.
+You take the next ticket from the backlog.
+Let's say it is called "resetting passwords".
+You run `git hack reset-passwords`. This command
+
+* checks out the _master_ branch
+* pulls updates for the _master_ branch, i.e. the things that Bob and Carol have shipped while you worked on your last feature
+* cuts a new feature branch called "password-reset" from your now up-to-date _master_ branch
+* checks out that new branch
+
+This gives you the best possible start for the new password reset feature.
+
+
+## Synchronizing the branch
+
+After coding for a while, you hear that Bob shipped a number of important bug fixes,
+and that Carol shipped some UI design updates.
+Both affect your work, so you want them in your branch before you continue working.
+In order to achieve that, you run `git sync`. This command
+
+* stashes away your currently open changes
+* checks out the _master_ branch
+* pulls the updates on that branch (Bob's bug fixes and Carol's UI updates)
+* checks out your feature branch again
+* merges the updates from the _master_ branch into your branch
+* pushes your updated feature branch to the repository
+* restores your open changes by popping the stash
```

Comment:
I kept it intentionally low-level because this tutorial is aimed at people who are new to Git Town. I want to give them a feeling for how complicated it is to use Git the right way in a realistic scenario, and how much Git Town helps here. Because too many people say "I can use normal Git directly without problem". That's why I listed the individual steps here. Stashing and checking out branches are an important part of that workflow, that's why I think its important to list them here. Sure, they are details, but that's stuff that a user without Git Town would have to type, so I would say its relevant here. 

You are right for the man pages/description of the individual commands, however.